src/ros2.cpp

```diff
@@ -380,7 +386,14 @@ QObject *Ros2QmlSingletonWrapper::createSubscription( const QString &topic,
 
 QObject *Ros2QmlSingletonWrapper::createServiceClient( const QString &name, const QString &type )
 {
-  return new ServiceClient( name, type );
+  return new ServiceClient( name, type, QoS( 1 ) );
```

Comment:
Not too happy about the magic `1` here, but the `QoS(..)` method below doesn't have a default value for the arg.

Could call `qml_ros2_plugin::QoS()` directly, but not sure that would be ok, stylistically.